packages/workbox-webpack-plugin/test/node/e2etest.js

```diff
@@ -140,6 +134,69 @@ describe(`Tests for webpack plugin`, function() {
   {
     "url": "styles/stylesheet-2.css",
     "revision": "\\w*"
+  },
+  {
+    "url": "webpackEntry.js",
+    "revision": "\\w*"
+  }
+\\]\\);`);
+      const result = regex.exec(swContents);
+      expect(result).to.exist;
+    });
+  });
+
+  it('should append publicPath to webpack generated files', () => {
+    fsExtra.copySync(
+      path.join(__dirname, '..', '..', '..', 'workbox-cli', 'test', 'static', 'example-project-1'),
+      tmpDirectory);
+
+    process.chdir(tmpDirectory);
+
+    const webpackConfig = {
+      entry: {
+        webpackEntry: path.join(tmpDirectory, 'webpackEntry.js'),
+      },
+      output: {
+        path: tmpDirectory,
+        filename: '[name].js',
+        publicPath: '/public',
```

Comment:
It could exclude a trailing `/`, since the Webpack documentation is just a recommendation and isn't enforced by Webpack's build process.

If you use `publicPath: '/public'` in this test case, then the URL that should be present in the precache manifest, as per Webpack's documented behavior, is `'/publicwebpackEntry.js'` (which obviously looks weird).